hyperspy/misc/eds/utils.py

```diff
@@ -243,6 +275,74 @@ def take_off_angle(tilt_stage,
                                   + math.sin(a) * math.sin(c)))
 
 
+def xray_lines_model(elements,
+                     beam_energy=200,
+                     weight_percents=None,
+                     energy_resolution_MnKa=130,
+                     energy_axis={'name': 'E', 'scale': 0.01, 'units': 'keV',
+                                  'offset': -0.1, 'size': 1024}
+                     ):
+    """
+    Generate a model of X-ray lines using a Gaussian distribution for each peak.
+
+    The area under a main peak (alpha) is equal to 1 and weighted by the
+    composition.
+
+    Parameters
+    ----------
+    elements : list of strings
+        A list of chemical element symbols.
+    beam_energy: float
+        The energy of the beam in keV.
+    weight_percents: list of float
+        The composition in weight percent.
+    energy_resolution_MnKa: float
+        The energy resolution of the detector in eV
+    energy_axis: dic
+        The dictionary for the energy axis. It must contains 'size' and the
+        units must be 'eV' of 'keV'.
+
+    Example
+    -------
+    >>> s = utils_eds.simulate_model(['Cu', 'Fe'], beam_energy=30)
+    >>> s.plot()
+    """
+    from hyperspy._signals.eds_tem import EDSTEMSpectrum
+    from hyperspy.model import components
+    s = EDSTEMSpectrum(np.zeros(energy_axis['size']), axes=[energy_axis])
+    s.set_microscope_parameters(
+        beam_energy=beam_energy,
+        energy_resolution_MnKa=energy_resolution_MnKa)
+    s.add_elements(elements)
+    counts_rate = 1.
+    live_time = 1.
```

Comment:
Sorry for the mixed in comments, I wasn't refreshing properly.

Regarding which value it should have: Looking at the full context of the code, these values (`live_time` and `count_rate`) should either be removed, or taken as arguments, as they currently have no function. If they are taken as arguments, they are currently used correctly to create the signal, but they should then be saved to the metadata, so that it will operate correctly afterwards. No?